### PR TITLE
Fix broken gzip stream

### PR DESCRIPTION
**Pull request by sigma-andex**
- Pipe input json into gzip stream

Solves #413

### CODE REVIEW

File: src/Pulp/Publish.purs [hunk 1]
```diff
@@ -16,6 +16,7 @@ import Data.Version (Version)
 import Data.Version as Version
 import Effect.Aff (Aff, attempt, throwError)
 import Effect.Class (liftEffect)
+import Effect.Class.Console (log)
```
Comment (JordanMartinez) [inline src/Pulp/Publish.purs]: Is this import needed?

**Review by JordanMartinez — APPROVED** [review]

File: src/Pulp/Publish.purs (hunk 1 above)
Comment (sigma-andex) [reply src/Pulp/Publish.purs]: Ah no, forgot to remove it.

**Comment by JordanMartinez** [comment]
Looks like you got some linting issues.

**Comment by sigma-andex** [comment]
> Looks like you got some linting issues.

whoops

**Comment by sigma-andex** [comment]
🐧 @JordanMartinez

**Review by JordanMartinez — APPROVED** [review]

**Comment by JordanMartinez** [comment]
> > Looks like you got some linting issues.
> 
> whoops

Ah, I interpreted this as "I need to push a fix" not a "I've pushed a fix." Ping me if I haven't responded in 3 days.

**Comment by JordanMartinez** [comment]
This has been published on NPM as `v16.0.2`.

**Pull request merged**